containers/podman-desktop
# chore: website update font and front page spacing

chore: website update font and front page spacing

### What does this PR do?

* Updates the font to Manrope
* Adjusts the font size and heights
* We add `leading-` twice due to:
  https://github.com/tailwindlabs/tailwindcss/discussions/5977 since it
  resets after you add `md:` or `lg:`
* Slightly adjusts font size when using a smaller screen

### Screenshot / video of UI



![Screenshot 2024-11-20 at 10 44 44 AM](https://github.com/user-attachments/assets/16c63cb3-e65a-49e1-ba6b-77a1532c67fe)
![Screenshot 2024-11-20 at 10 45 09 AM](https://github.com/user-attachments/assets/94801611-3ea4-4fe8-947e-6f5f6dc67bb6)



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop-internal/issues/376

### How to test this PR?



1. Launch the website
2. See new font

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
